tests/test_http_parser.py

```diff
@@ -117,6 +117,20 @@ def test_parse_headers(parser: Any) -> None:
     assert not msg.upgrade
 
 
+def test_parse_headers_longline(parser: Any) -> None:
+    text = (
+        b"""GET /test HTTP/1.1\r
+Test\xd9Header"""
+        + b"A" * 8192
+        + b": test\r"
+        + b"""\r
+\r
+"""
+    )
+    with pytest.raises(http_exceptions.LineTooLong):
```

Comment:
It is the best practice to always use the `match=` argument with `pytest.raises()` (and other similar helpers like `pytest.warns()` or `pytest.deprecated()`). Could you add it?